packages/react-divider/src/components/Divider/useDivider.ts

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { makeMergeProps, resolveShorthandProps } from '@fluentui/react-utilities';
+import { useMergedRefs } from '@fluentui/react-hooks';
```

Comment:
```suggestion
import { makeMergeProps, resolveShorthandProps, useMergedRefs } from '@fluentui/react-utilities';
```

Reply:
Im getting a no exported member for useMergeRefs from react-utilities ... I have the latest synced from this morning

Reply:
Must have been a vscode issue ... Updated!